### ISSUE TITLE
Adding new header file causes build to fail

### ISSUE THREAD

**Issue by kzawad1** [danielpinto8zz6/c-cpp-project-generator#159]
**Describe the bug**
Just adding one header file and including it in main.c causes a build error indicating that it cannot find the include file:
```
 >Executing task: bash -c make <

gcc -Wall -Wextra -g   -c -o src/main.o src/main.c
src/main.c:2:10: fatal error: test.h: No such file or directory
    2 | #include "test.h"
      |          ^~~~~~~~
compilation terminated.
make: *** [<builtin>: src/main.o] Error 1
The terminal process "/bin/bash '-c', 'bash -c make'" terminated with exit code: 2.

Terminal will be reused by tasks, press any key to close it.
```

**To Reproduce**
1. Open Visual Studio Code.
2. Create a new folder.
3. Open the folder in Visual Studio Code.
4. Run the C/C++ Project Generator.
5. Add a new empty header file named "test.h".
6. Add the header file in the main.c file using "#include "test.h"".
7. Trigger Terminal | Run Build Task.
8. Observe the error.

**Expected behavior**
Expecting the compile command to have a "-Iinclude" so that it knows where to find the header file:
`gcc -Wall -Wextra -g   -c -o src/main.o src/main.c`

**Screenshots**
![VisualStudioCode_BuildError](https://user-images.githubusercontent.com/13036629/95208113-4e0b4f00-07ae-11eb-8680-f4adc333b2c8.png)

**Environment**
 - VSCode Version: 1.49.2
 - OS Version: Ubuntu 20.04.1 LTS x64

**Additional context**
None

**Comment by kzawad1** [comment]
To get rid of the error with not being able to find include files that are located in the include folder, the **Makefile** can be modified in the following way:
```
all: $(BIN)/$(EXECUTABLE)
	@echo "Executing 'all' complete!"

.PHONY: clean
clean:
	-$(RM) $(BIN)/$(EXECUTABLE)
	-$(RM) $(OBJECTS)
	@echo "Cleanup complete!"


run: all
	./$(BIN)/$(EXECUTABLE)
	@echo "Executing 'run: all' complete!"

.c.o:
	$(CC) $(CFLAGS) $(CINCLUDES) -c $*.c -o $@
	@echo "Compiled "$<" successfully!"

$(BIN)/$(EXECUTABLE): $(OBJECTS)
	$(CC) $(CFLAGS) $(CINCLUDES) $(CLIBS) $^ -o $@ $(LIBRARIES)
	@echo "Linking complete!"
```

The C/C++ Project Generator template should be updated so that it is able to find the include files "right out of the box" without further modifications.

**Comment by danielpinto8zz6** [comment]
Hi, thanks for your contribution, I'll implement it in next release

**Comment by danielpinto8zz6** [comment]
https://github.com/danielpinto8zz6/c-cpp-project-generator/commit/687d4b7f6cfb6a599c69d2c5e04f04b7d26e9423

**Comment by NamidaSai** [comment]
Hi, I'm still getting the original error on the C++ template after the Makefile modifications.
Using the 1.2.2 release of cpp-project-generator and VS Code 1.50.1 on Windows 10. 
Any help appreciated!

**Comment by danielpinto8zz6** [comment]
@NamidaSai do you have the header files in include folder?

**Comment by NamidaSai** [comment]
Yes, i get the same error when the header files are placed in the include folder.

However, it builds fine when I place them in the src folder next to the source files.

**Comment by danielpinto8zz6** [comment]
@NamidaSai fixed in version 1.2.4
